csp_billing_adapter/local_csp.py

```diff
@@ -60,4 +60,8 @@ def get_csp_name(config: Config) -> str:
 @csp_billing_adapter.hookimpl(trylast=True)
 def get_account_info(config: Config) -> str:
     """Return the 'local' CSP's account info."""
-    return {'account_number': '123456789'}
+    return {
```

Comment:
Why do we return example data here?  Is the idea some plugin will "override" this so this will never be used? If so, maybe a comment is worth adding?

Reply:
That's just part of pluggy. All the functions are set to trylast=True. Thus the final plugin loaded that implements the hook will be run. In production only one plugin will be in an environment that would override this and that will get loaded after core. it's the same thing for every other hook. So adding a comment to just this one method doesn't seem ideal. Definitely something to document but probably in the core README wrt to deployment and plugin development.

Reply:
We can drop the local_csp and memory storage modules from loading by default. And they would just be in code as an example and for manually loading to run tests.

Reply:
I like that option.

Reply:
The memory storage modules are already dropped and only loaded in conftest.py now, and this module can be added to that list also. However for simulated deployment testing we would need to either extend the csp-billing-adapter-local plugin to register an appropriate hook implementation of this, or add a csp-billing-adapter-nocloud (or whatever we wanted to call it) that would provide an appropriate hook implementations for this, otherwise you would have noting implementing it...

It may also make sense to move the embedded "testing" hook implementations into a sub-package, e.g. csp_billing_adapter.testing, which hosts the memory_cache, memory_csp_usage, product_api, and local_csp plugins.

Reply:
Let's move this conversation to an issue. https://github.com/SUSE-Enceladus/csp-billing-adapter/issues/62